geps/gep-1897/index.md

```diff
@@ -170,18 +174,14 @@ as a TLS Client:
 
 - An explicit signal that TLS should be used by this connection.
 - A hostname the Gateway should use to connect to the backend.
-- A reference to one or more certificates to use in the TLS handshake, signed by a CA or self-signed.
-- An indication that system certificates may be used.
+- A reference to one or more CA certificates to validate the server's TLS certificates.
 
 BackendTLSPolicy is defined as a Direct Policy Attachment without defaults or overrides, applied to a Service that
 accesses the backend in question, where the BackendTLSPolicy resides in the same namespace as the Service it is
-applied to.  The BackendTLSPolicy and the Service must reside in the same namespace in order to prevent the
-complications involved with sharing trust across namespace boundaries.  We chose the Service resource as a target,
+applied to.  For now, the BackendTLSPolicy and the Service must reside in the same namespace in order to prevent the
+complications involved with sharing trust across namespace boundaries (see [Future plans](#future-plans)).  We chose the Service resource as a target,
 rather than the Route resource, so that we can reuse the same BackendTLSPolicy for all the different Routes that
 might point to this Service.
-For the use case where certificates are stored in their own namespace, users may create Secrets and use ReferenceGrants
-for a BackendTLSPolicy-to-Secret binding.  Implementations must respect a ReferenceGrant for cross-namespace Secret
-sharing to BackendTLSPolicy, even if they don't for other cross-namespace sharing.
```

Comment:
Why remove this?

Reply:
This is not a part of the current implementation, so I was aligning the GEP to reality.

Reply:
How do you know that none of the 5 implementations implemented this?

Reply:
It is impossible. The CRD does not have a "namespace" field at all.